src/WebAppDIRAC/WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js

```diff
@@ -282,7 +251,7 @@ Ext.define("DIRAC.JobMonitor.classes.JobMonitor", {
       ["site", "site"],
       ["status", "status"],
       ["types", "jobType"],
-      ["OwnerGroup", "OwnerGroup"],
```

Comment:
I'm not sure why `OwnerGroup` is being removed?

Reply:
It is replaced by VO

Reply:
indeed, restored.